pkgs/os-specific/linux/kernel/linux-zen.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, fetchFromGitHub, buildLinux, ... } @ args:
 
 let
-  version = "5.8.13";
+  version = "5.9.0";
```

Comment:
Tag is `5.9`, not `5.9.0`

Reply:
If I set it to `5.9` it fails to build since `modDirVersion` doesn't match.

Reply:
I can separate them into different variables I guess. I had set it to `5.9` initially and found this afterwards.

Reply:
What I did was to just append `.0` in modDirVersion (and add a comment).

Perhaps we should make the kernel build able to handle major.minor though.